EncoreTickets.ConsoleTester/Program.cs

```diff
@@ -14,24 +14,41 @@ class Program
     {
         static void Main(string[] args)
         {
-            ApiContext context = new ApiContext(Environments.Sandbox);
-            List<string> productIds = new List<string>();
-            context.affiliate = "encoretickets";
-            // uscontext.useBroadway = true;
+            var context = new ApiContext(Environments.Sandbox, "admin", "password") {affiliate = "encoretickets"};
```

Comment:
why can't the context.userName and context.password be part of the Environments.Sandbox? or args

Reply:
Because Environments is enum in this project and is used only to set the type of environment.

Reply:
so how are we gonna pass different username/pass per integration? cos if I understand correctly, this is hardcoded